src/components/popover.vue

```diff
@@ -33,7 +33,7 @@ implementation of the Bootstrap plugins. We could also consider implementing our
 own popover functionality, perhaps using Popper.
 -->
 <template>
-  <div class="popover-content-source">
+  <div class="popover-content-source" aria-hidden="true">
```

Comment:
Because we are now displaying this off-screen rather than hiding it with `display: none`, we tell screen readers to ignore it.